editor.planx.uk/src/ui/editor/TemplatedNodeConfiguration.tsx

```diff
@@ -0,0 +1,84 @@
+import EditNoteIcon from "@mui/icons-material/EditNote";
+import Typography from "@mui/material/Typography";
+import type { TemplatedNodeData } from "@opensystemslab/planx-core/types";
+import { BaseNodeData } from "@planx/components/shared";
+import { useFormik } from "formik";
+import React from "react";
+import Input from "ui/shared/Input/Input";
+import InputRow from "ui/shared/InputRow";
+import { Switch } from "ui/shared/Switch";
+
+import InputGroup from "./InputGroup";
+import InputLabel from "./InputLabel";
+import ModalSection from "./ModalSection";
+import ModalSectionContent from "./ModalSectionContent";
+
+export type TemplatedNodeConfigurationProps<T extends BaseNodeData> = {
+  formik: ReturnType<typeof useFormik<T>>;
+  disabled?: boolean;
+} & TemplatedNodeData;
+
+export const TemplatedNodeConfiguration = <T extends BaseNodeData>({
+  formik,
+  isTemplatedNode,
+  templatedNodeInstructions,
+  areTemplatedNodeInstructionsRequired,
+  disabled,
+}: TemplatedNodeConfigurationProps<T>) => {
+  return (
+    <ModalSection>
+      <ModalSectionContent title="Templates" Icon={EditNoteIcon}>
```

Comment:
Do we want to lean into the star icon here or happy with this `EditNoteIcon`?

Reply:
Swapped for :star: based on feedback in this thread https://opensystemslab.slack.com/archives/C07F7215W7P/p1749479047827879